adoc/Private-Registry-Installation.adoc

```diff
@@ -459,9 +465,9 @@ persistence:
       secretkey: <aws-account-secret-key>
 ----
 
-. (Optional) Configure a GitHub authentication token for trivy
+. (Optional) Configure a GitHub authentication token for Trivy
 +
-If the `trivy` security vulnerability scanner service is enabled, it is recommended to link:https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token[generate a GitHub personal authentication token] and supply it in the `harbor-values.yaml` trivy configuration section, to prevent issues with link:https://docs.github.com/en/free-pro-team@latest/rest/reference/rate-limit[the API rate-limiting that github enforces on unauthenticated requests]:
+If the `Trivy` security vulnerability scanner service is enabled, we recommend link:https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token[generating a GitHub personal authentication token] and supplying it in the `harbor-values.yaml` trivy configuration section, to prevent issues with link:https://docs.github.com/en/free-pro-team@latest/rest/reference/rate-limit[the API rate-limiting that github enforces on unauthenticated requests]:
```

Comment:
Typo:

... [the API rate-limiting that GitHub enforces on unauthenticated requests]: